docs/designers-developers/developers/data/data-core-block-editor.md

```diff
@@ -115,6 +115,19 @@ _Returns_
 
 -   `number`: Number of blocks in the post.
 
+<a name="getBlockDropTarget" href="#getBlockDropTarget">#</a> **getBlockDropTarget**
```

Comment:
I guess the changes on this file are a  leftover?

Reply:
Oh yep, left behind from an abandoned iteration.